do-not-commit/formatted-string-bashoperator.py

```diff
@@ -0,0 +1,81 @@
+from datetime import timedelta
+
+import requests
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+from airflow.utils.dates import days_ago
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": days_ago(2),
+    "email": ["airflow@example.com"],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+}
+
+dag = DAG(
+    "tutorialex2",
+    default_args=default_args,
+    description="Tutorial DAG",
+    schedule_interval=timedelta(days=1),
+)
+
+message = requests.get("https://fakeurl.asdf/message").text
+# ruleid: formatted-string-bashoperator
+t1 = BashOperator(task_id="print_date", bash_command="echo " + message, dag=dag)
+
+howlong = requests.get("https://fakeurl.asdf/howlong").text
```

Comment:
By default, 'requests' calls wait until the connection is closed.
This means a 'requests' call without a timeout will hang the program
if a response is never received. Consider setting a timeout for all
'requests'. ([actice.use-timeout.use-timeout](https://semgrep.dev/r/actice.use-timeout.use-timeout))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=actice.use-timeout.use-timeout) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>